third_party/move/move-compiler-v2/tests/checking/inlining/lambda_param.exp

```diff
@@ -11,3 +11,6 @@ error: Invalid call target: currently indirect call must be a parameter to an in
   │
```

Comment:
This seems like a duplicate error: would the two errors here (at line 7 above and this one) always be coupled (i.e., always appear in pairs)? If so, we could remove reporting the error with "indirect call" and only keep the one above?

Reply:
Ok, done.